api/get_rbg_hls_link.php

```diff
@@ -1,20 +1,26 @@
 <?php
 include 'simple_html_dom.php';
-include_once 'CacheAccess.php';
 
 function get_rbg_hls_link(string $url): ?string {
```

Comment:
A function comment explaining what the "hls link" is, would greatly increase the readability.